postcss.config.js

```diff
@@ -1,15 +1,9 @@
-const presetEnv = require('postcss-preset-env')
-const extend = require('postcss-extend')
-const nano = require('cssnano')
-
 module.exports = {
-  map: true,
-  plugins: [
-    presetEnv({
+  plugins: {
+    'postcss-preset-env': {
       stage: 0,
-    }),
-    extend(),
-    nano({
+    },
+    'cssnano': {
```

Comment:
If you configure it so, there is no need to use `.js`. You can make this file `.yml`, `.json` or `rc`

Reply:
I think nextjs supports only [.js or .json](https://nextjs.org/docs/advanced-features/customizing-postcss-config) which is probably the same.